.buildkite/packaging.pipeline.yml

```diff
@@ -26,7 +25,6 @@ steps:
   - name: Start of concurrency group for DRA Staging
     # exceptionally allow building staging from main when VERSION_QUALIFIER is set, to allow prerelease testing
     # TODO remove OR clause below and above comment, and only allow matching /^[0-9]+\.[0-9x]+\$/ for build.branch
-    if: build.branch =~ /^[0-9]+\.[0-9x]+\$/ || build.env('VERSION_QUALIFIER') != null
     command: echo "--> Start of concurrency gate dra-staging"
     concurrency_group: "dra-gate-staging-$BUILDKITE_BRANCH"
     concurrency: 1
```

Comment:
DRA build failed with

```
Rename dependencies to 9.1.0-SNAPSHOT
mv: cannot stat 'build/distributions/dependencies.csv': No such file or directory
```

you'll either need to pass

```
RUN_STAGING="true"
```

or comment out the `if` clause (e.g. in line 41 below) for snapshot + staging to enable `- label: Snapshot dashboards` and `- label: Staging dashboards` to be allowed to run and produce the expected csv file.